util/session/sessionmanager.go

```diff
@@ -278,6 +256,18 @@ func (mgr *SessionManager) Parse(tokenString string) (jwt.Claims, error) {
 		return token.Claims, nil
 	}
 
+	capability := settings.AccountCapabilityApiKey
+	if parts := strings.Split(subject, ":"); len(parts) > 1 {
```

Comment:
I think the format of how we formulate the subject convention is too deeply hidden in the code. Can we describe the subject convention somewhere, or better yet, have utility methods in jwtutil to parse the capabilities from a token so that the interpretation of subject is at a central place?

Reply:
done